src/Microsoft.IdentityModel.Tokens/JsonWebKey.cs

```diff
@@ -143,12 +142,6 @@ private void Copy(JsonWebKey key)
         /// </summary>
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore, NullValueHandling = NullValueHandling.Ignore, PropertyName = JsonWebKeyParameterNames.E, Required = Required.Default)]
         public string E { get; set; }
-        /// <summary>
-        /// Gets or sets the 'k' (Symmetric - Key Value)..
-        /// </summary>
-        /// Base64urlEncoding
-        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore, NullValueHandling = NullValueHandling.Ignore, PropertyName = JsonWebKeyParameterNames.K, Required = Required.Default)]
-        public string K { get; set; }
```

Comment:
I am thinking we leave the key as is and support Symmetric. The SignatureProvider can examine and return the appropriate provider.